src/main/java/se/sundsvall/emailsender/api/model/SendEmailRequest.java

```diff
@@ -22,47 +24,77 @@
 @Builder(setterPrefix = "with", toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor(access = AccessLevel.PACKAGE)
+@Schema(description = "Send e-mail request")
 public class SendEmailRequest {
-    
-	@Schema(description = "Recipient e-mail address", example = "recipient@recipient.se" )
-    @NotBlank
-    @Email
-    private String emailAddress;
-
-    @Schema(description = "E-mail subject")
-    @NotBlank
-    private String subject;
-
-    @Schema(description = "E-mail plain-text body")
-    private String message;
-
-    @Schema(description = "E-mail HTML body (BASE64-encoded)")
-    @ValidBase64(nullable = true)
-    private String htmlMessage;
-
-    @Valid
-    @NotNull
-    private Sender sender;
-
-    @Schema(description = "Attachments")
-    private List<@Valid Attachment> attachments;
-
-    @Getter
-    @NoArgsConstructor
-    @AllArgsConstructor(access = AccessLevel.PACKAGE)
-    @Builder(setterPrefix = "with")
-    public static class Attachment {
-
-        @Schema(description = "The attachment (file) content as a BASE64-encoded string", example = "aGVsbG8gd29ybGQK")
-        @ValidBase64
-        private String content;
-
-        @Schema(description = "The attachment filename", example = "test.txt")
-        @NotBlank
-        private String name;
-
-        @Schema(description = "The attachment content type", example = "text/plain")
-        @NotBlank
-        private String contentType;
-    }
+
+	@Schema(description = "Recipient e-mail address", example = "recipient@recipient.se")
+	@NotBlank
+	@Email
+	private String emailAddress;
+
+	@Schema(description = "E-mail subject")
+	@NotBlank
+	private String subject;
+
+	@Schema(description = "E-mail plain-text body")
+	private String message;
+
+	@Schema(description = "E-mail HTML body (BASE64-encoded)")
+	@ValidBase64(nullable = true)
+	private String htmlMessage;
+
+	@Schema(description = "Sender")
```

Comment:
`@Schema` for the sender exists both here and on the actual sender class below. You should settle for one or the other